gapipy/resources/dossier/activity_dossier.py

```diff
@@ -23,3 +24,42 @@ class ActivityDossier(Resource):
         ('start_location', 'Place'),
         ('end_location', 'Place'),
     ]
+
+    _model_collection_fields = [
+        ('details', DossierDetail),
+    ]
+
+    def __repr__(self):
+        return '<{} {}>'.format(self.__class__.__name__, self.name)
+
+    @property
+    def duration(self):
+        """
+        Simulate the duration dict in `ItineraryComponent` objects,
```

Comment:
@Nagyman Yes. I feel it would be an obvious mistake in most implementations, e.g. they would be expecting to get max_hr via `dossier.duration.max_hr` rather than `dossier.duration['max_hr']`